helm/orca/config/alerts-mapping.yaml

```diff
@@ -185,6 +185,416 @@ prometheus:
+          namespace: namespace
+    - name: PrometheusNotificationQueueRunningFull
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusErrorSendingAlertsToSomeAlertmanagers
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusErrorSendingAlertsToAnyAlertmanager
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusNotConnectedToAlertmanagers
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusDuplicateTimestamps
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusMissingRuleEvaluations
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusNotIngestingSamples
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusOutOfOrderTimestamps
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusRemoteStorageFailures
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusRemoteWriteBehind
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusRemoteWriteDesiredShards
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusRuleFailures
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusTSDBCompactionsFailing
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusTSDBReloadsFailing
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: pod
+          namespace: namespace
+    - name: PrometheusOperatorReconcileErrors
+      source_mapping:
+        origin: kubernetes
+        kind: namespace
+        properties:
+          name: namespace
+    - name: PrometheusOperatorNodeLookupErrors
+      source_mapping:
+        origin: kubernetes
+        kind: namespace
```

Comment:
Let's map to cluster kind instead of namespace kind.

Namespace kind is not enabled in the graph - we should avoid it as much as possible. Since many other Prometheus alerts (e.g. `PrometheusOperatorDown`) have been already mapped to cluster kind, we should be consistent.